data_collection/gazette/spiders/ma_imperatriz.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date
```

Comment:
Recomendaria não importar diretamente o `date` pois é muito utilizado na base de código também para representar a data de publicação dos diários. Também diria o mesmo sobre o `parse` mas outro comentário abordar o `dateparser`
```suggestion
import datetime
```